tests/staker_test.go

```diff
@@ -25,18 +25,24 @@ func FuzzTestStakerDelegationsWithPaginationResponse(f *testing.F) {
 	attachRandomSeedsToFuzzer(f, 3)
 	f.Fuzz(func(t *testing.T, seed int64) {
```

Comment:
Why instead of creating a new test we modify an existing one to make it test a different test case?